packages/node_modules/pouchdb-generate-replication-id/src/index.js

```diff
@@ -35,6 +35,11 @@ function generateReplicationId(src, target, opts) {
   return Promise.all([src.id(), target.id()]).then(function (res) {
     var queryData = res[0] + res[1] + filterFun + filterViewName +
       queryParams + docIds + selector;
+      
+    if (opts.attachments === false) {
+      queryData += 'attachments:false';
```

Comment:
if ('attachments' in opts) {
      queryData += 'attachments:' + opts.attachments;

We probably arent going to be changing defaults or anything here, but best practise just in case we would